### PR TITLE
Audit fix: HashConsensus

### PR DESCRIPTION
**Pull request by bulbozaur**

### CODE REVIEW

File: contracts/committees/HashConsensus.sol [hunk 1]
```diff
@@ -101,10 +107,16 @@ abstract contract HashConsensus is Ownable {
         view
         returns (uint256 support, uint256 executionQuorum, Timestamp scheduledAt, bool isUsed)
     {
-        support = _getSupport(hash);
-        executionQuorum = quorum;
-        scheduledAt = _hashStates[hash].scheduledAt;
-        isUsed = _hashStates[hash].usedAt.isNotZero();
+        HashState storage hashState = _hashStates[hash];
```
Comment (Psirex) [inline contracts/committees/HashConsensus.sol:110]: It may be read in `memory` as it fits in one slot

File: contracts/committees/HashConsensus.sol [hunk 2]
```diff
@@ -62,8 +64,12 @@ abstract contract HashConsensus is Ownable {
         approves[msg.sender][hash] = support;
         emit Voted(msg.sender, hash, support);
 
-        if (_getSupport(hash) == quorum) {
+        uint256 currentSupport = _getSupport(hash);
+
+        if (currentSupport >= quorum) {
```
Comment (Psirex) [inline contracts/committees/HashConsensus.sol:69]: Consider adding a test that checks this edge case

**Review by Psirex — APPROVED** [review]
